src/domain/app/i18n/fi.json

```diff
@@ -1716,6 +1721,44 @@
     "warningNoRightsToCreate": "Sinulla ei ole oikeuksia luoda paikkoja.",
     "warningNoRightsToEdit": "Sinulla ei ole oikeuksia muokata tätä paikkaa."
   },
+  "priceGroup": {
+    "deletePriceGroupModal": {
+      "buttonDelete": "Poista hintaryhmä",
```

Comment:
I think previously we decided to use the term "asiakasryhmä" in all three languages. Probably should be used in the administration UI too.

Reply:
Replaced "hintaryhmä" with the term "asiakasryhmä" in all three languages.